docs/conf.py

```diff
@@ -4,14 +4,33 @@
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
+import re
+
 from rocm_docs import ROCmDocs
 
+with open('../CMakeLists.txt', encoding='utf-8') as f:
+    match = re.search(r'set \( VERSION_STRING \"?([0-9.]+)[^0-9.]+', f.read())
+    if not match:
+        raise ValueError("VERSION not found!")
+    version_number = match[1]
+left_nav_title = f"hipSOLVER {version_number} Documentation"
 
-external_projects_current_project = "hipsolver"
+# for PDF output on Read the Docs
+project = "hipSOLVER Documentation"
+author = "Advanced Micro Devices, Inc."
+copyright = "Copyright (c) 2023 Advanced Micro Devices, Inc. All rights reserved."
+version = version_number
+release = version_number
+
+external_toc_path = "./sphinx/_toc.yml"
 
-docs_core = ROCmDocs("hipSOLVER Documentation")
-docs_core.run_doxygen()
+docs_core = ROCmDocs(left_nav_title)
+docs_core.run_doxygen(doxygen_root="doxygen", doxygen_path="doxygen/xml")
+docs_core.enable_api_reference()
 docs_core.setup()
+docs_core.setup()
```

Comment:
It looks like the setup call is duplicated? Also, I didn't see `enable_api_reference` in https://github.com/ROCmSoftwarePlatform/rocSOLVER/pull/626. Should it be present there as well?

Reply:
Thanks for catching that. Since hipSOLVER documentation does not use [doxysphinx](https://pypi.org/project/doxysphinx/) these 2 extra lines are not needed.

```
docs_core.enable_api_reference()
docs_core.setup()
```